### DISCUSSION TITLE
Ability to suppress Wayland warning message

### DISCUSSION THREAD

**Discussion by mrguitar** [input-leap/input-leap#1944] [Feature Requests]
Hey,
I think the current warning message for the status of Wayland support is approximate. Honestly, it works well enough for my daily use, and it would be *soooooooo* nice if I can pass a flag to suppress that warning message. Every time I see it I get annoyed. :) ....but I think it should stay on by default. A tiny fix here would be nice and hopefully super easy. Cheers and thank you so much for keeping this project going. It's awesome.

**Comment by shymega** [comment]
There is a PR for this. I would like to keep it on by default, and add a way to suppress it.

Waiting for review.
